src/Symfony/Component/Translation/Writer/TranslationWriter.php

```diff
@@ -88,4 +88,19 @@ public function writeTranslations(MessageCatalogue $catalogue, $format, $options
         // save
         $dumper->dump($catalogue, $options);
     }
+
+    /**
+     * @param MessageCatalogue $catalogue The message catalogue to write
+     * @param string           $format    The format to use to dump the messages
+     * @param array            $options   Options that are passed to the dumper
+     *
+     * @throws InvalidArgumentException
+     *
+     * @deprecated since 3.4 will be removed in 4.0. Use write instead.
+     */
+    public function writeTranslations(MessageCatalogue $catalogue, $format, $options = array())
+    {
+        @trigger_error('Method writeTranslations() is deprecated. Use write() instead.', E_USER_DEPRECATED);
```

Comment:
`@trigger_error(sprintf('Method %s() is deprecated since version 3.4 and will be removed in 4.0. Use write() instead.', __METHOD__), E_USER_DEPRECATED);`